src/Lean/Parser/LeanToken.lean

```diff
@@ -0,0 +1,326 @@
+import Lean.Parser.Basic
+
+/-! Tokenization functions used for the implementation of the Lean language. -/
```

Comment:
Have you just moved the code? Or, have you made modifications too?

Reply:
Only changes for moving functions from `ParserFn` to `TokenParserFn`, plus https://github.com/Kha/lean4/commit/bd44113a58f385e687a5beb12f20c6ff29257a24